server/hap2/hatohol/hap2_nagios_ndoutils.py

```diff
@@ -182,12 +194,16 @@ def collect_triggers_and_put(self, fetch_id=None, host_ids=None):
             in_cond = "','".join(host_ids)
             sql += " WHERE %s.host_object_id in ('%s')" % (t2, in_cond)
 
-        # NOTE: The update time in the output is renewed every status
-        #       check in Nagios even if the value is not changed.
-        # TODO by 15.09:
-        #   We should has the previous result and compare it here in order to
-        #   improve performance. Or other columns such as last_state_change in
-        #   nagios_servicestatus might be used.
+        update_type = "ALL"
+        if fetch_id is None:
```

Comment:
The purpose of this condition should be indicated in order to improve readability.

For exampe.

def all_triggers_should_send():
    return fetch_id is None

if all_triggers_should_send():
   ....